operating/manage-vulnerabilities/vulnerability-management.adoc

```diff
@@ -47,26 +55,10 @@ include::modules/vulnerability-management-review-deferred.adoc[leveloffset=+3]
 
 include::modules/vulnerability-management-reporting.adoc[leveloffset=+2]
 
-[id="vulnerability-reporting-vuln-mgt1"]
-== Vulnerability reporting in Vulnerability Management 1.0
-
-The following sections provide instructions for creating vulnerability reports in Vulnerability Management 1.0. Beginning with version 4.2, {product-title-short} provides Vulnerability Reporting 2.0 (Technology Preview), which includes enhanced vulnerability reporting features. Instructions for using Vulnerability Reporting 2.0 are provided in a later section.
-
-include::modules/vulnerability-management-scheduled-report.adoc[leveloffset=+2]
-
-include::modules/vulnerability-management-send-report.adoc[leveloffset=+2]
-
-include::modules/vulnerability-management-edit-report.adoc[leveloffset=+2]
-
-include::modules/vulnerability-management-delete-report.adoc[leveloffset=+2]
-
 [id="vulnerability-reporting-vuln-mgt2"]
-== Vulnerability reporting in Vulnerability Management 2.0
+== Vulnerability reporting
 
-Vulnerability reporting in Vulnerability Management 2.0 provides the ability to create and download an on-demand image vulnerability report. This report includes a comprehensive list of common vulnerabilities and exposures across images and deployments, called workload CVEs in {product-title-short}. You can share this report with auditors or internal stakeholders by scheduling emails in {product-title-short} or by downloading the report and sharing it by using other methods.
-
-:FeatureName: Vulnerability Reporting 2.0
-include::snippets/technology-preview.adoc[]
+From the *Vulnerability Management (2.0)* menu, you can create and download an on-demand image vulnerability report. This report includes a comprehensive list of common vulnerabilities and exposures across images and deployments, called workload CVEs in {product-title-short}. You can share this report with auditors or internal stakeholders by scheduling emails in {product-title-short} or by downloading the report and sharing it by using other methods.
```

Comment:
I'd mention where this menu is. Presumably it's the same console throughout, but it's not clear out of context.